src/configurations/destinations/intercom_v2/db-config.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "INTERCOM_V2",
+  "displayName": "Intercom V2",
+  "config": {
+    "transformAtV1": "router",
+    "saveDestinationResponse": true,
+    "throttlingCost": { "eventType": { "group": 3, "identify": 3 } },
+    "auth": {
+      "type": "OAuth",
+      "role": "intercom",
+      "rudderScopes": ["delivery"]
+    },
+    "includeKeys": ["oneTrustCookieCategories", "consentManagement", "ketchConsentPurposes"],
+    "excludeKeys": [],
+    "supportedSourceTypes": [
+      "android",
+      "ios",
+      "web",
+      "unity",
+      "amp",
+      "cloud",
+      "reactnative",
+      "flutter",
+      "cordova",
+      "shopify"
+    ],
+    "supportedMessageTypes": {
+      "cloud": ["identify", "track", "group"]
+    },
+    "supportedConnectionModes": {
+      "web": ["cloud"],
+      "android": ["cloud"],
+      "ios": ["cloud"],
+      "unity": ["cloud"],
+      "amp": ["cloud"],
+      "reactnative": ["cloud"],
+      "flutter": ["cloud"],
+      "cordova": ["cloud"],
+      "shopify": ["cloud"],
+      "cloud": ["cloud"]
+    },
+    "destConfig": {
+      "defaultConfig": ["rudderAccountId", "apiServer", "sendAnonymousId"],
+      "android": [
```

Comment:
useNativeSDK should not be required - @sandeepdsvs can you confirm ?

Reply:
removed this after confirming with @sandeepdsvs